xrmocap/data/dataset/builder.py

```diff
@@ -4,13 +4,16 @@
 from .base_dataset import BaseDataset
 from .mview_mperson_dataset import MviewMpersonDataset
 from .mvp_dataset import MVPDataset
+from .fourd_dataset import FourDDataset
 
 # yapf: enable
 
 DATASETS = Registry('dataset')
 DATASETS.register_module(
     name='MviewMpersonDataset', module=MviewMpersonDataset)
 DATASETS.register_module(name='MVPDataset', module=MVPDataset)
+DATASETS.register_module(
+    name='FourDDataset', module=FourDDataset)
```

Comment:
I suggest we use BottomUpMviewMpersonDataset here. Maybe we will rename MviewMpersonDataset to TopDownMviewMpersonDataset one day.